packages/request-handler/src/releases-providers.ts

```diff
@@ -67,12 +63,7 @@ export const Providers: ProvidersProps = {
           ),
         })
         .transform((resp) => {
-          return resp.results.map((release) => ({
-            ...release,
-            releaseUrl: "",
-            releaseDescription: "",
-            isPreRelease: false,
-          }));
+          return resp.results.map((release) => release);
```

Comment:
Isn't this redundant?
```suggestion
          return resp.results;
```